src/CMakeLists.txt

```diff
@@ -481,7 +478,7 @@ endif(${HAVE_ARMV8_CRC})
 
 add_library(common_utf8 STATIC common/utf8.c)
 
-target_link_libraries(common json_spirit common_utf8 erasure_code rt uuid ${CRYPTO_LIBS} ${Boost_LIBRARIES} ${BLKID_LIBRARIES} ${EXECINFO_LIBRARIES})
```

Comment:
@tchaikov What was the reasoning behind taking erasure_code out of common? In src/common/Makefile.am $LIBERASURE_CODE is part of LIBCOMMON_DEPS